src/Microsoft.ML.Data/Transforms/ColumnSelecting.cs

```diff
@@ -682,7 +682,7 @@ public Row GetRow(Row input, Func<int, bool> active)
                 return new RowImpl(input, _mapper);
             }
 
-            public IDataTransform ApplyToData(IHostEnvironment env, IDataView newSource)
+            IDataTransform ITransformTemplate.ApplyToData(IHostEnvironment env, IDataView newSource)
```

Comment:
>ITransformTemplate [](start = 27, length = 18)

I agree with you that this is probably the best policy since it even discourages internal code from using this now deprecated idiom, but just in case you are curious, it is not necessary to do this here since this nested class (`SelectColumnsDataTransform`) is itself not public.